webservices/resources/reports.py

```diff
@@ -46,25 +50,35 @@ def parse_types(types):
             'type': 'string',
         },
         {
+
             'name': 'committee_type',
             'in': 'path',
             'type': 'string',
-            'description': 'House, Senate or presidential',
-            'enum': ['presidential', 'pac-party', 'house-senate'],
+            'description': 'House, Senate, presidential, independent expenditure only',
+            'enum': ['presidential', 'pac-party', 'house-senate', 'ie-only'],
         },
     ],
 )
 class ReportsView(Resource):
 
+    @schemas.marshal_with(schemas.CommitteeReportsPageSchema(), wrap=False)
```

Comment:
This is what I am thinking-

There are different types of form 5- the ones that I am dealing with here are the Quarterly and Year-end reports, similar in concept to the form 3, 3x and 3p Because they cover a period of time rather than transactions. 

Here is the [form 5](http://www.fec.gov/pdf/forms/fecfrm5.pdf) for reference.

This endpoint doesn't have all the form 5s, The 24 and 42 hour notices will go in a separate independent expenditures endpoint, like you suggested, like the Form 3 details that are in the schedules, will go in a separate endpoint. 

That way, all the summary reports are covered by a /reports endpoint and you can find them with committee/id/reports without having to look up what the committee type is.

We could try and put them all into one schema with nulls  or n/a for fields that don't apply so it works in Swagger, but  I don't want to burden users with having to understand the different kinds of summary reports and where to look for each kind because, the independent expenditure totals for the other groups is on the form 3x.